README.md

```diff
@@ -81,3 +75,1052 @@ And the run it the domain you would like to taget:
 ```shell
 ostorlab scan run --install -g agent_group.yaml domain-name example.com
 ```
+
+
+## Current Coverage
+
+For the moment, we are currently focused on the CISA KEV Database and Google Tsunami.
+
+| CVE ID           | Implemented | Note                                              |
+| ---------------- | ----------- | ------------------------------------------------- |
+| CVE-2002-0367    | ❌           | Not remotely exploitable/User interaction needed. |
+| CVE-2004-0210    | ❌           | Not remotely exploitable/User interaction needed. |
+| CVE-2004-1464    | ❌           | Missing public exploit                            |
```

Comment:
```suggestion
| CVE-2004-1464    | ❌           | Missing public exploit.                           |
```
Please respect punctuation in all the sentences.

Reply:
done